NHentai/sync/infra/adapters/repositories/hentai/implementations/nhentai.py

```diff
@@ -10,7 +10,7 @@
 
 class NHentaiAdapter(NhentaiInterface):
 
-    _BASE_URL = 'https://nhentai.net/'
+    _BASE_URL = 'https://translate.google.com/translate?sl=fil&tl=en&hl=fil&u=https://nhentai.net/&client=webapp'
```

Comment:
use "format" to be able to re-use this constant.

_BASE_URL = 'https://translate.google.com/translate?sl=fil&tl=en&hl=fil&u=https://nhentai.net{path}&client=webapp'

self._BASE_URL.format('/random')

request_response = self.request_adapter.get(self._BASE_URL.format('/random'))